### DISCUSSION TITLE
Conditional "pnpm.overrides" ?

### DISCUSSION THREAD

**Discussion by rasgo-cc** [pnpm/pnpm#8514] [General]
Hi,

I have a monorepo with some libs I want to reuse on other projects. Since everything is in the early stages and updates are constant (and nothing is in production) I'm not using any registry to publish these packages. On projects that need those libs I'm using `pnpm.overrides` to point to those libs which are on my computer.

However, once I go into production of course this won't work and I'll have to publish packages. 
So the question is:

Is there a way to conditionally use `pnpm.overrides` so I'd use them (or some of them at least) locally during development?

From what I was able to understand, I believe the answer here is "no" but in that case, what would you suggest to handle this scenario?

Thank you!

**Comment by zkochan** [comment]
No, there are no such "dev only" overrides. Just make your changes to overrides and don't commit them.
